src/Nri/Ui/QuestionBox/V3.elm

```diff
@@ -309,10 +321,19 @@ viewGuidance config referencingId markdown_ =
                 ]
 
         Nothing ->
-            viewSpeechBubble config
-                referencingId
-                [ Balloon.markdown markdown_
-                , Balloon.css [ Css.margin2 (Css.px 10) (Css.px 20) ]
+            div
+                [ css
+                    [ Css.displayFlex
+                    , Css.justifyContent Css.flexEnd
+                    , Css.position Css.relative
+                    ]
+                ]
```

Comment:
How would you feel about adding the new content in higher in the DOM instead of down in viewGuidance? I'm thinking maybe [here](https://github.com/NoRedInk/noredink-ui/blob/master/src/Nri/Ui/QuestionBox/V3.elm#L272), in `viewBalloon`? I think that way, no new wrapping position-relative divs need to be added.

Reply:
Makes sense @tesk9, just did it.